pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "pathfinder-monster-creator"
+version = "0.1.0"
+description = "Project supporting pen & paper RPG game design using machine learning."
+authors = [
+    "Paulina Jędrychowska <>",
+    "Bogumiła Papiernik <>",
+    "Jolanta Śliwa <>"
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+pandas = "1.4.*"
+numpy = "1.21.*"
+scikit-learn = "1.2.*"
```

Comment:
Najlepiej trzymać to posortowane leksykograficznie, ułatwia pracę, jak zaczyna być tego więcej